src/CMakeLists.txt

```diff
@@ -359,24 +359,76 @@ endif()
 
 include(../3rdparty/cmake-scripts/sanitizers.cmake)
 
+if(WITH_QT6)
 find_package(
-  Qt5 5.14 REQUIRED
+  Qt6 6.2.0 REQUIRED
   COMPONENTS Core
              Multimedia
              Network
              OpenGL
              UiTools
              Widgets
-             Concurrent)
-find_package(Qt5 COMPONENTS Gamepad QUIET)
-find_package(Qt5 COMPONENTS TextToSpeech QUIET)
+             Concurrent
+             Core5Compat)
+  find_package(Qt6 COMPONENTS Gamepad QUIET)
+  find_package(Qt6 COMPONENTS TextToSpeech QUIET)
+
+  message(STATUS "Qt version: ${Qt6Core_VERSION}")
+
+  set(QT_MAJOR_VERSION 6)
+  set(QT_LIBS Qt6::Concurrent
+              Qt6::Core
+              Qt6::Network
+              Qt6::Multimedia
+              Qt6::OpenGL
+              Qt6::UiTools
+              Qt6::Widgets
+              Qt6::Core5Compat
+              qt6keychain)
+
+  if(Qt6TextToSpeech_FOUND)
+    list(APPEND QT_LIBS Qt6::TextToSpeech)
+    message(STATUS "Using TextToSpeech module")
+  endif()
 
-if(Qt5Core_VERSION VERSION_LESS 5.14)
-  message(
-    FATAL_ERROR "Mudlet requires Qt 5.14 or later, yours is ${Qt5Core_VERSION}")
+  if(Qt6Gamepad_FOUND)
```

Comment:
I think this is pointless - they have dropped the QGamepad module in Qt 6. See: https://wiki.qt.io/Qt_6.0.0_Modules .

Reply:
Ah, missed that change. 

Will remove this check for now, as it doesn't look this is coming back anytime [soon](https://lists.qt-project.org/pipermail/development/2021-March/041125.html).

Reply:
Added another suggested follow-up to the PR description.